docs/src/index.md

```diff
@@ -492,7 +492,97 @@ end
 end
 ```
 
-# Working with `Symbol`s without referring to columns
+## Constructing multi-column arguments and `src => fun => dest` calls using `$`
+
+If an argument is entirely wrapped in `$()`, the result bypasses the anonymous function creation of DataFramesMeta.jl and is passed to the underling DataFrames.jl function directly. Importantly, this allows for multi-column selection and passing `src => fun => dest` calls from the DataFrames.jl "mini-language" directly.
+
+### Multi-argument selectors 
+
+To refer to multiple columns in DataFrames.jl, one can write
+
+```julia
+select(df, [:a, :b])
+```
+
+which selects the columns `:a` and `:b` in the data frame. We can generate this command in DataFramesMeta.jl with
+
+```julia
+@select df $[:a, :b]
+```
+
+Similarly, to selct all columns beginning with the letter `"a"`, wrap a regular expression in `$()`. 
+
+```julia
+@select df $(r"^a")
```

Comment:
why are `(` and `)` needed here and were not needed above?

Reply:
Added an explanation here and above.